AppCenterAuth/AppCenterAuth/Internals/Models/MSAuthAuthority.h

```diff
@@ -5,7 +5,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface MSIdentityAuthority : NSObject
+@interface MSAuthAuthority : NSObject
```

Comment:
`MSAuthConfigAuthority` maybe? Not a blocker.

Reply:
I think we should rename this, otherwise it could be very confusing.

Reply:
This file name is the biggest concern that I have. Thanks for feedback!
However we need a discussion about the name for the class.

Reply:
It will be `MSAuthority`.